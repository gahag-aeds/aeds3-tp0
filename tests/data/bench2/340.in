340 170
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 
Add 221 333
Sum 212 324
Add 137 249
Min 176 288
Sub 101 213
Sum 17 129
Sub 171 283
Max 61 173
Add 84 196
Min 115 227
Add 225 337
Min 177 289
Add 192 304
Min 103 215
Add 166 278
Min 2 114
Sub 17 129
Sum 157 269
Add 14 126
Max 147 259
Sub 46 158
Max 20 132
Add 162 274
Min 112 224
Sub 81 193
Sum 141 253
Sub 57 169
Max 209 321
Add 152 264
Sum 109 221
Add 3 115
Min 58 170
Add 6 118
Sum 51 163
Sub 6 118
Max 106 218
Sub 67 179
Sum 176 288
Sub 78 190
Max 151 263
Add 64 176
Max 75 187
Sub 12 124
Min 167 279
Sub 89 201
Min 178 290
Sub 80 192
Max 105 217
Add 107 219
Min 93 205
Add 163 275
Min 64 176
Sub 25 137
Sum 97 209
Add 88 200
Max 17 129
Add 149 261
Min 144 256
Add 226 338
Min 212 324
Add 165 277
Max 140 252
Sub 42 154
Sum 82 194
Add 190 302
Max 186 298
Sub 187 299
Sum 168 280
Sub 47 159
Max 176 288
Sub 3 115
Min 22 134
Sub 162 274
Max 15 127
Sub 100 212
Sum 23 135
Sub 104 216
Min 180 292
Add 39 151
Sum 122 234
Add 45 157
Max 201 313
Sub 98 210
Max 208 320
Add 209 321
Min 185 297
Add 225 337
Max 42 154
Add 13 125
Max 135 247
Add 165 277
Min 89 201
Add 47 159
Max 119 231
Add 170 282
Min 148 260
Add 77 189
Min 41 153
Sub 227 339
Sum 35 147
Sub 128 240
Sum 3 115
Add 56 168
Sum 62 174
Add 156 268
Max 156 268
Add 223 335
Min 32 144
Sub 20 132
Min 34 146
Add 65 177
Sum 64 176
Add 146 258
Max 162 274
Sub 184 296
Max 39 151
Add 119 231
Sum 181 293
Add 219 331
Max 44 156
Sub 227 339
Min 157 269
Sub 132 244
Max 185 297
Sub 48 160
Sum 214 326
Sub 105 217
Max 36 148
Sub 27 139
Max 104 216
Sub 210 322
Sum 155 267
Sub 18 130
Max 38 150
Sub 128 240
Sum 173 285
Sub 105 217
Sum 124 236
Add 204 316
Sum 124 236
Add 69 181
Min 42 154
Add 100 212
Max 127 239
Add 115 227
Sum 56 168
Sub 77 189
Max 7 119
Add 148 260
Max 69 181
Add 189 301
Min 147 259
Sub 225 337
Sum 5 117
Add 104 216
Min 184 296
Sub 218 330
Min 120 232
Sub 131 243
Max 18 130
Sub 135 247
Max 113 225
Add 84 196
Max 153 265
Add 151 263
Min 211 323
Add 98 210
Max 167 279
Sub 19 131
Max 214 326
