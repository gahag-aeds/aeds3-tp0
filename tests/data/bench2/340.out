5850
1
5347
99
0
0
0
0
6274
101
98
1
6230
102
5665
-1
5090
101
6245
103
100
0
-1
98
-2
-4
5432
99
0
4
107
5332
108
6859
107
-7
98
4667
1
5760
106
106
5
94
107
-4
105
-3
-2
5008
5221
5058
110
-1
-2
5260
112
98
6822
98
2
111
6711
97
103
6821
95
6641
5727
5760
-4
107
4913
99
104
0
4993
5
1
95
106
111
4
112
112
